app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
The Python community does not recommend using BaseException for custom Exceptions. This can cause implicit problems in certain situations. It is recommended to use the Exception class